application/classes/Ushahidi/Core.php

```diff
@@ -162,8 +167,11 @@ public static function init()
 
 		// Post Exporter
 		$di->setter['Ushahidi\Console\Application']['injectCommands'][] = $di->lazyNew('Ushahidi_Console_PostExporter');
+		$di->setter['Ushahidi_Console_PostExporter']['setDatabase'] = $di->lazyGet('kohana.db');
 		$di->setter['Ushahidi_Console_PostExporter']['setPostExportRepo'] = $di->lazyGet('repository.posts_export');
 		$di->setter['Ushahidi_Console_PostExporter']['setDataFactory'] = $di->lazyGet('factory.data');
+		$di->setter['Ushahidi_Console_PostExporter']['setPostDataExportRepo'] = $di->lazyGet('repository.postdataexport');
```

Comment:
sorry deleted my previous comment, I mean to clean up a bit more in my head . `MUST READ BEFORE HITTING BUTTON` x 100

Reply:
the gist of it was: 
`TLDR: This is an export, this makes sense as an export. But the posts export we HAD, that one is no longer an export. Does that make sense? `